graphql/schema/User.ts

```diff
@@ -89,10 +99,14 @@ export const LoginMutation = mutationField("login", {
 			},
 		});
 
+		const PREVENT_TIMING_ATTACKS = await hashPassword(
+			"preventTimingAttacks1234",
+		);
+
 		if (
 			!(await verifyPassword(
 				password,
-				user?.password ?? "preventTimingAttacks123",
+				user?.password ?? PREVENT_TIMING_ATTACKS,
```

Comment:
This is a critical vulnerability! It allows the (publicized) password "preventTimingAttacks123" to be used to impersonate anyone.